test/test_routes.py

```diff
@@ -39,4 +39,20 @@ def client():
 
 def test_latest_article(client):
     response = client.get("/latest-article")
-    assert response.json == {"response": "OK"}
\ No newline at end of file
+    assert response.json == {"response": "OK"}
+
+
+@app.route('/not-found')
+def testing():
+    raise MissingUserError()
+
+ext = JSONExceptionHandler(app)
+ext.register(code_or_exception=MissingUserError)
+
+with app.app_context():
+    with app.test_client() as c:
+        rv = c.get('/not-found')
+
+assert rv.status_code == 404
```

Comment:
These asserts are present at the file level instead of being in a method. What are you trying to achieve through these couple of lines?